Extension/src/LanguageServer/Providers/workspaceSymbolProvider.ts

```diff
@@ -30,9 +30,9 @@ export class WorkspaceSymbolProvider implements vscode.WorkspaceSymbolProvider {
             let name: string = symbol.name;
             if (suffix.length) {
                 if (symbol.scope === SymbolScope.Private) {
-                    suffix = "private" + localize("c.cpp.workspacesymbolscope.separator", ", ") + suffix;
+                    suffix = localize("c.cpp.symbolscope.separator", "{0}, {1}", "private", suffix);
```

Comment:
You might want to add a 'hint' here for translators, who might not understand what they should do with `{0}, {1}`.

Adding a hint would look something like:

```
suffix = localize({ key: "c.cpp.symbolscope.separator", comment: ["Used in a list as a separator between keywords.  Example: ..."] }, "{0}, {1}", "private", suffix);
```

Reply:
Sure, I wasn't sure if "hint" was available or not yet.